quiz1.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title></title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://kit.fontawesome.com/630f3095fb.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://practical-poitras-d410a0.netlify.app/packages/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quintessential&family=Roboto:ital,wght@0,300;1,400&family=Zen+Kurenaido&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="navbar-title">
+            quizzy
+        </div>
+        <div>
+            <a href="index.html" class="navbar-link">Home</a>
+        </div>
+        <div class="navbar-items">
+
+            <a href="login.html" class="btn btn-primary">Login</a>
+        </div>
+    </nav>
+    <div class="container">
+        <div class="grid grid-1-3 display-center">
+            <div>
+                <div class="card">
+                    <div class="card-text">
+                        <div class="flex my-s"><span>Question: 1/5</span> <span>score: 0</span></div>
```

Comment:
what does the "my-s" class do here?

suggestion: class names can be more descriptive